aws-lambda/aws-lambda.d.ts

```diff
@@ -1,55 +1,61 @@
 // Type definitions for AWS Lambda
 // Project: http://docs.aws.amazon.com/lambda
-// Definitions by: Michael Skarum <https://github.com/skarum>
+// Definitions by: James Darbyshire <https://github.com/darbio/aws-lambda-typescript>, Michael Skarum <https://github.com/skarum>, Stef Heyenrath <https://github.com/StefH/DefinitelyTyped>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 declare module "aws-lambda" {
-
-    export interface Records {
-        Records: Record[];
-    }
-    interface Record {
-        EventVersion: string;
-        EventSubscriptionArn: string;
-        EnventSource: string;
-        Sns: SNS;
-        kinesis: Kinesis;
-    }
-    interface SNS {
-        Type: string;
-        MessageId: string;
-        TopicArn: string;
-        Subject: string;
-        Message: string;
-        Timestamp: Date;
-    }
-
-    interface Kinesis {
-        data: string;
-    }
-
-    export interface Context {
-        log(message: string, object: any): void;
```

Comment:
If you remove log(), fail() and succeed() you break compatability with Lambda running node 0.1
Many still uses the old version due to some issues: http://stackoverflow.com/questions/37466847/dynamodb-causing-lambda-timeout